sklearnex/tests/utils/base.py

```diff
@@ -369,3 +372,41 @@ def _get_processor_info():
         )
 
     return proc
+
+
+class DummyEstimator(BaseEstimator):
+
+    def fit(self, X, y=None):
+        sua_iface, xp, _ = _get_sycl_namespace(X)
+        X_table = to_table(X)
+        y_table = to_table(y)
+        # The presence of the fitted attributes (ending with a trailing
+        # underscore) is required for the correct check. The cleanup of
+        # the memory will occur at the estimator instance deletion.
+        if sua_iface:
+            self.x_attr_ = from_table(
+                X_table, sua_iface=sua_iface, sycl_queue=X.sycl_queue, xp=xp
+            )
+            self.y_attr_ = from_table(
+                y_table, sua_iface=sua_iface, sycl_queue=X.sycl_queue, xp=xp
```

Comment:
Is it Ok that `y_attr_` goes with X's queue?

Also, what happens if `X` and `y` are from different namespaces and have different `sua_iface`? For example, `X` - from dpnp and `y` - from numpy. What is expected to happen in this case?

```suggestion
                y_table, sua_iface=sua_iface, sycl_queue=y.sycl_queue, xp=xp
```

Reply:
This was a failure in the original implementation, though the backend in sklearnex is very fuzzy in this (not standard). For questions about from_table I would ask @samir-nasibli.